### PR TITLE
Added notes about Vert.x removal in the CHANGELOG

### PR DESCRIPTION
**Pull request by ppatierno**
Trivial PR updating the CHANGELOG with main Vert.x removal stuff.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 * Fixed regression about producing messages not working when CORS is enabled
 * Dependency updates (Netty 4.1.86.Final)
+* Refactored producer, consumer and admin endpoints by using the Java native Kafka client instead of the Vert.x one
```
[the remark this answers is not in the document]
Comment (ppatierno) [reply CHANGELOG.md:7]: Totally agree. I have never liked saying "native" but didn't come with a good term. I will use "Apache Kafka client APIs" then.

**Review by scholzj — APPROVED** [review]

**Review by im-konge — APPROVED** [review]

**Comment by ppatierno** [comment]
The additional commits also fix the failing tests related to a "zombie" instance of the admin client facade trying to connect to an old instance of the test-container Kafka cluster.

**Comment by scholzj** [comment]
You probably should have renamed the PR - or ideally opened a separate PR. It is quite messy to have bunch of code changed in a commit / PR which says _update CHANGELOG_.

**Comment by ppatierno** [comment]
Just renamed it now. It's too late. I was trying to fix the tests where they were failing so working on the same PR.

**Comment by scholzj** [comment]
Yeah, but the commit is already merged.

**Pull request merged**